com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/persistence/query/ConfigurationEntryQuery.java

```diff
@@ -19,11 +20,23 @@
     ConfigurationEntryQuery requiredProperties(Map<String, Object> requiredProperties);
 
     ConfigurationEntryQuery spaceId(String spaceId);
+    
+    ConfigurationEntryQuery spaceIdNotNull();
```

Comment:
All new methods except for `deleteAll` aren't used anywhere (at least as far as I can see). Can we remove them from the PR?